src/pages/ServiceDetails/ServiceInfo/__tests__/__snapshots__/ServiceInfoDescription.test.tsx.snap

```diff
@@ -137,6 +137,7 @@ exports[`#ServiceInfoDescription render correctly with data should render servic
                 />
               </StackItem>
             </Stack>
+            33
```

Comment:
Snatshop typo.

I think most of our snapshots tests are a little bit outdated and the only test they perform is "it renders with some basic info".

But nothing to do with this PR, just this is an example about the "outdated" status of these ones.